indico/web/client/js/tinymce.js

```diff
@@ -5,6 +5,9 @@
 // modify it under the terms of the MIT License; see the
 // LICENSE file for more details.
 
+import './prism';
```

Comment:
I wasn't sure where to put the the library files - maybe `web/static` if we can import from there?

Reply:
can't we install it via npm?

Reply:
I'm looking into it now, it seemed more complex at a first glance so I went with this first